xml/art_sle_ha_install_quick.xml

```diff
@@ -184,9 +271,13 @@
     </listitem>
    </varlistentry>
    <varlistentry xml:id="vle.ha.req.ssh">
-    <term>SSH <remark>toms, 2016-07-30: (kai) ha_join takes care of SSH</remark></term>
+    <term>SSH</term>
     <listitem>
      &sys-req-other-ssh;
+     <para>
+      If you use the <command>ha-cluster-init</command> script, you will be
```

Comment:
As both ha-cluster-init and ha-cluster-join configure SHH keys, better rephrase to: "If you use the bootstrap scripts for setting up the cluster, SSH keys will automatically be created and copied.